.github/actions/composite/updateProtectedBranch/action.yml

```diff
@@ -0,0 +1,140 @@
+name: Update Protected Branch
```

Comment:
Reviewer note: this action is largely the same as the deleted `.github/workflows/updateProtectedBranch.yml`, so comparing it to that might make it easier to review.